scripts/common/repository.js

```diff
@@ -55,7 +55,7 @@ class MonoRepository {
       });
   }
 
-  static release(version, repos) {
+  static release(version, repos, dryRelease) {
```

Comment:
suggestion: 
```suggestion
  static release(version, repos, dryRelease = false) {
```
(even though with JS `!dryrelease` will result with the same)